### PR TITLE
Feature/apps 3229 internal dashboard

### PR DESCRIPTION
**Pull request by solomontang**
- Global dashboard initial implementation
  - See design here https://wiki.openlattice.com/display/MET/Chronicle+Internal+Metrics
- Show Summary stats, count of total organizations, studies, and participants.
- Tabular data that shows brief overview of each study and it's participants. This can take a while to load until a dedicated API is created.
image

### CODE REVIEW

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]
Please remove everything after line 19 in `flow-typed/npm/@fortawesome/pro-duotone-svg-icons_vx.x.x.js`

File: config/webpack/webpack.config.base.js [hunk 1]
```diff
@@ -134,10 +134,6 @@ module.exports = (env) => {
         'core-js/library/fn/array/includes': path.resolve(NODE, 'core-js-pure/features/array/includes'),
       },
       extensions: ['.js', '.css'],
-      modules: [
-        SOURCE,
-        NODE,
-      ],
```
Comment (UnsungHero97) [inline config/webpack/webpack.config.base.js:140]: I was able run the app locally without having to make this change. I don't think this is necessary.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/core/router/DefaultUnauthorized.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { faLockAlt } from '@fortawesome/pro-duotone-svg-icons';
+// @flow
+import { IconSplash, Spinner } from 'lattice-ui-kit';
```
Comment (UnsungHero97) [inline src/core/router/DefaultUnauthorized.js]:
```suggestion
// @flow

import { faLockAlt } from '@fortawesome/pro-duotone-svg-icons';
import { IconSplash, Spinner } from 'lattice-ui-kit';
```

File: src/containers/dashboard/utils/generateTableData.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import padStart from 'lodash/padStart';
+import random from 'lodash/random';
+// @flow
+import range from 'lodash/range';
+import { DateTime } from 'luxon';
+import { v4 as uuid } from 'uuid';
```
Comment (UnsungHero97) [inline src/containers/dashboard/utils/generateTableData.js]:
```suggestion
// @flow

import padStart from 'lodash/padStart';
import random from 'lodash/random';
import range from 'lodash/range';
import { DateTime } from 'luxon';
import { v4 as uuid } from 'uuid';
```

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: config/webpack/webpack.config.base.js (hunk 1 above)
Comment (solomontang) [reply config/webpack/webpack.config.base.js:140]: Interesting. I was not... I had an issue where parts of nivo dependencies would not resolve correctly so while the app was built and launched, viewing the charts would crash

Comment (UnsungHero97) [reply config/webpack/webpack.config.base.js:140]: Hmmm ... let me try that.

Comment (UnsungHero97) [reply config/webpack/webpack.config.base.js:140]: I cannot reproduce, neither with npm v7 nor npm v8.

**Review by anzioka — DISMISSED** [review]

File: config/webpack/webpack.config.base.js (hunk 1 above)
Comment (solomontang) [reply config/webpack/webpack.config.base.js:140]: not crashing anymore, but warnings are still issued that are resolved with this change

**Review by UnsungHero97 — DISMISSED** [review]

**Review by bergieboy — DISMISSED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**